packages/wonder-blocks-dropdown/components/option-item.js

```diff
@@ -148,9 +159,9 @@ export default class OptionItem extends React.Component<OptionProps> {
                                 selected={selected}
                                 {...state}
                             />
-                            <LabelLarge style={styles.label}>
+                            <LabelMedium style={styles.label}>
                                 {label}
-                            </LabelLarge>
+                            </LabelMedium>
```

Comment:
Why did the size of the label change?  Is there a JIRA ticket for this?

Reply:
Ohh good call, I didn't mention that here. We found a visual bug with the option items using `font-weight: bold` instead of `normal`. As the change was too small, I decided to make it as part of this PR.

Something I didn't think before, is that `LabelLarge` and `LabelMedium` have the same `font-size` but different `font-weight`. Maybe we should revisit this as the names could be suggesting different sizes between them.